### PR TITLE
467 shared hosting text

### LINKED ISSUE #467: Revise hosting provider page top matter to be more clear
Based on the latest round of user testing.

top

### PR DESCRIPTION
**Pull request by lschatzkin**
closes #467

### CODE REVIEW

File: hosting_providers/index.md [hunk 1]
```diff
@@ -4,25 +4,14 @@ title: Does My Hosting Provider Offer HTTPS?
...
-</ol>
-
-## Do you use shared hosting?
-
-If you’re not using the shared hosting product, or if your provider doesn’t appear here, check out our [instructions](/instructions) for using Certbot or [get help](/help).
+<div class="shared-hosting">
+  <h3>This page is for users of {% include tooltip.html term-name="shared-hosting" text="shared hosting" %}.</h3>
+  <br>
+
+  If you're not using the shared hosting product, or if your provider doesn't appear here, check out our <a href="/instructions">instructions</a> for using Certbot or <a href="/help">get help</a>. Note that your {% include tooltip.html term-name="shared-hosting" text="shared hosting provider" %} is not the same as your {% include tooltip.html term-name="domain-registrar" text="domain registrar" %}.
```
Comment (wioux) [inline hosting_providers/index.md]: What do you think about nixing the second glossary link for shared-hosting here?

Comment (lschatzkin) [reply hosting_providers/index.md]: I think it'd be useful to include the sentence "Note that your shared hosting provider
is not the same as your domain registrar.", without re-highlighting "shared hosting". Lots of newbies conflate the two.

Comment (wioux) [reply hosting_providers/index.md]: I just mean not making the second use of the term link to the glossary again.

Comment (lschatzkin) [reply hosting_providers/index.md]: Oh, okay!

**Pull request merged**